components/CatCard.tsx

```diff
@@ -230,19 +229,24 @@ export const CatCard = ({
                 </View>
               </View>
             )}
-            {!isEmpty(cat.media) && (
-              <FlatList
-                style={{ marginTop: 20 }}
-                horizontal
-                data={[] /*cat.media*/}
-                keyExtractor={(item, index) => index.toString()}
-                renderItem={({ item }) => (
-                  <View>
-                    <Image style={styles.media} source={{ uri: item }} />
-                  </View>
-                )}
-              />
-            )}
+            {/* {!isEmpty(cat.media) && ( */}
```

Comment:
please remove unused code

Reply:
at this moment is unused, but we need it when we are working with the real data